app/helpers/application_helper.rb

```diff
@@ -9,10 +9,19 @@ def show_title?
     @show_title
   end
 
+  def uswds_class_for flash_type
+    { success: "success", error: "danger", alert: "warning", notice: "info" }[flash_type] || flash_type.to_s
+  end
+
   def flash_messages(opts = {})
     flash.each do |msg_type, message|
-      concat(content_tag(:div, content_tag(:div, :class=>"usa-alert-body" ), :class =>"usa-alert usa-alert-error")  do
-          concat message
+      concat(
+      content_tag(:div, :class=>"usa-alert usa-alert-#{uswds_class_for(msg_type)}") do
```

Comment:
small detail, but this is what is know as "old hash syntax" - can we move to "new hash syntax" in this entire method?

`class: 'usa-alert'`

instead of 

`:class => 'usa-alert'`